src/vstest.console/Resources/xlf/Resources.cs.xlf

```diff
@@ -1645,6 +1645,11 @@
         <target state="translated">    Přeskočené: {0}</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="InvalidArgument">
+        <source>The argument {0} provided is incorrect or invalid. Kindly use /help option to check the valid args. Know more at : https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options</source>
```

Comment:
The argument {0} is invalid. Please use the /help option to check the list of valid arguments.

Reply:
Updated error message